funding/manager.go

```diff
@@ -1306,9 +1306,10 @@ func (f *Manager) handleFundingOpen(peer lnpeer.Peer,
 		if err != nil {
 			log.Errorf("unable to query wallet: %v", err)
 		}
+		err := errors.New("Synchronizing blockchain")
```

Comment:
nit: change the commit message to `funding+lnwire: remove ErrSynchronizingChain `

Reply:
Made the change, thanks